### PR TITLE
Bump babel

### PR DESCRIPTION
**Pull request by yucohen**
bumped babel version
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9632

### CODE REVIEW

File: Tests/scripts/validate.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ if [[ $CI_COMMIT_BRANCH = master ]] || [[ -n "${NIGHTLY}" ]] || [[ -n "${BUCKET_
         PACKS_TO_UPLOAD_SPACED=${PACKS_TO_UPLOAD//,/ }
         for item in $PACKS_TO_UPLOAD_SPACED; do
             python3 -m demisto_sdk validate -i Packs/"$item" --post-commit --graph --skip-pack-dependencies
-        done       
+        done
```
Comment (yucohen) [inline Tests/scripts/validate.sh:12]:
```suggestion
        done    
```

File: Tests/scripts/validate.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ if [[ $CI_COMMIT_BRANCH = master ]] || [[ -n "${NIGHTLY}" ]] || [[ -n "${BUCKET_
...
```
Comment (yucohen) [inline Tests/scripts/validate.sh:12]:
```suggestion
        done    
```

Comment (yucohen) [inline Tests/scripts/validate.sh:12]:
```suggestion
        done     
```

**Review by tkatzir — APPROVED** [review]

**Review by JudahSchwartz — APPROVED** [review]
🎸 🚀

**Pull request merged**